e2e/dashboard-new-layouts/dashboard-group-panels.spec.ts

```diff
@@ -0,0 +1,152 @@
+import { e2e } from '../utils';
+
+describe('Grouping panels', () => {
+  beforeEach(() => {
+    e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
+  });
+
+  after(() => {
+    e2e.flows.revertAllChanges();
+  });
+
+  it('can group and ungroup new panels into row', () => {
+    e2e.flows.addDashboard({ title: 'Group new panels into row' });
+    cy.contains('Group new panels into row').should('be.visible');
+
+    // Toggle edit mode
+    e2e.components.NavToolbar.editDashboard.editButton().should('be.visible').click();
```

Comment:
i think there is now a flow for this. i think it's more of a self describing function so you don't need the comment, and it simplifies this line a bit